tasks/update-code/git.yml

```diff
@@ -29,6 +29,12 @@
   register: ansistrano_git_result
   when: ansistrano_git_identity_key_path|trim != ""
 
+- name: ANSISTRANO | GIT | Shred GIT deployment key
+  command: shred -f git_identity_key
+  args:
```

Comment:
Would prefer to avoid chdir by using `shred -f {{ ansistrano_deploy_to }}/git_identity_key`